resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/Config.scala

```diff
@@ -257,10 +257,10 @@ private[spark] object Config extends Logging {
 
   val KUBERNETES_DRIVER_SUBMIT_CHECK =
     ConfigBuilder("spark.kubernetes.submitInDriver")
-    .internal()
-    .version("2.4.0")
-    .booleanConf
-    .createWithDefault(false)
+      .internal()
+      .version("2.4.0")
+      .booleanConf
+      .createWithDefault(false)
```

Comment:
I think it's not worthwhile as it will cause a conflict against old branches when you backport stuff. Maybe let's fix it together when you happen to fix codes around here.

Reply:
Sounds reasonable. Thanks for your review! I would like close this PR and fix it in next related PR.